Cesium.IntegrationTests/stdio_test.c

```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
+
+int main()
+{
+    puts("test");
```

Comment:
Probably this fails to parse due to string literals not being supported, yet.

Reply:
I will play. Maybe I will change test somehow

Reply:
There nothing which I can do. I try following variants with same error in compiler, so I probably have to remove this file from integration tests.
```
//char t[5] = { 't', 'e', 's', 't', '\0', };
char t[5] = { 0x74, 0x65, 0x73, 0x74, 0, };
puts(t);
//puts("test");
```

Reply:
All of these aren't supported, yet :)

I propose to put this aside for now (a couple of days max; probably a dozen of hours) and give me some time to implement at least `const char* val = "hello world";`

Reply:
I think if I can add some function which does not rely on char, maybe math functions and test these parts of runtime lib. That way we will move forward.

Reply:
That would be okay, yeah. Feel free to omit this particular integration test for now.